### DISCUSSION TITLE
Make legacy talk.pimcore.org available again

### DISCUSSION THREAD

**Discussion by solverat** [pimcore/pimcore#7704] [General]
Using githubs discussions feature is not a bad idea at all. But now, all topics (https://talk.pimcore.org/) are gone. Any plans to migrate them? Or providing an archive?

**Comment by brusch** [comment]
Migrating doesn't seem to be possible at the moment. But we're trying to answer the top 20 questions in our docs (makes sense anyway).

**Comment by NiklasBr** [comment]
GitHub Feedback [Possibility to "import" data from other forum platform?](https://github.com/github/feedback/discussions/36):

> we would like to provide this option to you but currently do not have an API publicly available yet

**Comment by esisl** [comment]
Why did they kill the Talks?
Do you think all users should die next?
Has the board of directors changed?

**Comment by brusch** [comment]
@esisl we have decided to use GitHub Discussions in the future, as it makes sense to have this all in one place, also in terms of cross referencing topics and so on. I think this decision is understandable and that the community as a whole will benefit from this change. 

If you have arguments against this decision, please share, we are and were always open to a neat discussion 😊

**Comment by esisl** [comment]
It makes no difference on which platform to place the talk
But WHY, to destroy the entire accumulated database of questions and answers ???!!

Is it possible to lay out everything at least in flat text somewhere in an accessible place?

**Comment by brusch** [comment]
@esisl we're working on that

**Comment by brusch** [comment]
Here we go 😊
https://talk.pimcore.org/

**Comment by esisl** [comment]
https://talk.pimcore.org/ redirected 2 https://github.com/pimcore/pimcore/discussions

**Reply by dvesh3** [reply]
please refresh your browser cache 😉

**Comment by esisl** [comment]
"I'm not crazy" (c) :)

No, it seems to have stuck "somewhere there." We will wait.

**Reply by HaaseIT** [reply]
Some browsers (eg. Firefox) cache redirects. A quick web-search about the topic should give you the required steps.

**Comment by esisl** [comment]
Sadly, I have to keep 5 types of browsers open at the same time :D

However, through the proxy, it became clear.

**Comment by solverat** [comment]
@brusch Perfect! Thanks!
